README.md

```diff
@@ -51,8 +40,28 @@ Image security policies define Portieris' behavior in your cluster. There are tw
 
 ## Configuring image security policies
 
-You can configure custom security policies to control what images can be deployed in your Kubernetes namespaces, and to enforce trust pinning of particular signers. For more information, see the [IBM Cloud docs](https://console.bluemix.net/docs/services/Registry/registry_security_enforce.html#customize_policies).
+You can configure custom security policies to control what images can be deployed in your Kubernetes namespaces, and to enforce trust pinning of particular signers.  
+Portieris supports sourcing trust data from the following registries without additional configuration in the image policy:
+* IBM Cloud Container Registry
+* Quay.io
+* Docker Hub
 
+If users would like to use a different trust server for a repository, you can do so by configuring the policy:  
```

Comment:
We tend to use second person "you", rather than third person "users". Could this be written as below?

> To use a different trust server for a repository, you can specify the `trustServer` parameter in your policy:

Reply:
Done